bin/metadata_validation.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+import json
+
+import synapseclient
+from synapseclient import Entity
+
+
+def get_synaspe_file(syn_id):
+    """
+    Initializes synapseclient session and retrieves file metadata
+    """
+    syn = synapseclient.Synapse()
+    syn.login(authToken=os.environ["SYNAPSE_AUTH_TOKEN"])
+    syn_file = syn.get(syn_id)
+    return syn_file
+
+def json_dump(syn_id, test, data):
```

Comment:
**Nit (_i.e._ [optional](https://google.github.io/eng-practices/review/reviewer/standard.html#:~:text=Reviewers%20should%20always%20feel%20free,they%20could%20choose%20to%20ignore.)):** I would clarify the second argument as `test_name`.